pulseguardian/model/pulse_user.py

```diff
@@ -27,16 +27,19 @@ class PulseUser(Base):
         Queue, backref='owner', cascade='save-update, merge, delete')
 
     @staticmethod
-    def new_user(username, password='', owner=None, management_api=None):
+    def new_user(username, password='', owner=None, skip_management=False):
```

Comment:
Good call.  Yeah, I wasn't quite sure what the right naming should be there, but this is clearer.  Going with your idea.